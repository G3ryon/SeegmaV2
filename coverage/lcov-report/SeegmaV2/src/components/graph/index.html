
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for SeegmaV2/src/components/graph</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> SeegmaV2/src/components/graph</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">2.54% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>3/118</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/38</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/48</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">2.63% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>3/114</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line low'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file low" data-value="graph.js"><a href="graph.js.html">graph.js</a></td>
	<td data-value="2.04" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 2%"></div><div class="cover-empty" style="width: 98%"></div></div>
	</td>
	<td data-value="2.04" class="pct low">2.04%</td>
	<td data-value="49" class="abs low">1/49</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="10" class="abs low">0/10</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="18" class="abs low">0/18</td>
	<td data-value="2.13" class="pct low">2.13%</td>
	<td data-value="47" class="abs low">1/47</td>
	</tr>

<tr>
	<td class="file low" data-value="graphDisp.js"><a href="graphDisp.js.html">graphDisp.js</a></td>
	<td data-value="3.7" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 3%"></div><div class="cover-empty" style="width: 97%"></div></div>
	</td>
	<td data-value="3.7" class="pct low">3.7%</td>
	<td data-value="27" class="abs low">1/27</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="12" class="abs low">0/12</td>
	<td data-value="4" class="pct low">4%</td>
	<td data-value="25" class="abs low">1/25</td>
	</tr>

<tr>
	<td class="file low" data-value="settings.js"><a href="settings.js.html">settings.js</a></td>
	<td data-value="2.38" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 2%"></div><div class="cover-empty" style="width: 98%"></div></div>
	</td>
	<td data-value="2.38" class="pct low">2.38%</td>
	<td data-value="42" class="abs low">1/42</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="24" class="abs low">0/24</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="18" class="abs low">0/18</td>
	<td data-value="2.38" class="pct low">2.38%</td>
	<td data-value="42" class="abs low">1/42</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Thu Mar 18 2021 09:42:36 GMT+0100 (GMT+01:00)
            </div>
        </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    