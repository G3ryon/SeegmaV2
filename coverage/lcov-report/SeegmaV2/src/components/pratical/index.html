
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for SeegmaV2/src/components/pratical</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> SeegmaV2/src/components/pratical</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">80.18% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>89/111</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">50% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>35/70</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">83.02% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>44/53</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">83.84% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>83/99</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line high'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file empty" data-value="charts.js"><a href="charts.js.html">charts.js</a></td>
	<td data-value="0" class="pic empty">
	<div class="chart"><div class="cover-fill" style="width: 0%"></div><div class="cover-empty" style="width: 100%"></div></div>
	</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file medium" data-value="datumPicker.js"><a href="datumPicker.js.html">datumPicker.js</a></td>
	<td data-value="50" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 50%"></div><div class="cover-empty" style="width: 50%"></div></div>
	</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="6" class="abs medium">3/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="3" class="abs low">1/3</td>
	<td data-value="60" class="pct medium">60%</td>
	<td data-value="5" class="abs medium">3/5</td>
	</tr>

<tr>
	<td class="file high" data-value="input.js"><a href="input.js.html">input.js</a></td>
	<td data-value="86.36" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 86%"></div><div class="cover-empty" style="width: 14%"></div></div>
	</td>
	<td data-value="86.36" class="pct high">86.36%</td>
	<td data-value="22" class="abs high">19/22</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="10" class="abs medium">5/10</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="8" class="abs high">8/8</td>
	<td data-value="88.89" class="pct high">88.89%</td>
	<td data-value="18" class="abs high">16/18</td>
	</tr>

<tr>
	<td class="file high" data-value="modal.js"><a href="modal.js.html">modal.js</a></td>
	<td data-value="83.33" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 83%"></div><div class="cover-empty" style="width: 17%"></div></div>
	</td>
	<td data-value="83.33" class="pct high">83.33%</td>
	<td data-value="6" class="abs high">5/6</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	<td data-value="80" class="pct high">80%</td>
	<td data-value="5" class="abs high">4/5</td>
	<td data-value="83.33" class="pct high">83.33%</td>
	<td data-value="6" class="abs high">5/6</td>
	</tr>

<tr>
	<td class="file high" data-value="radioButton.js"><a href="radioButton.js.html">radioButton.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="6" class="abs high">6/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	</tr>

<tr>
	<td class="file high" data-value="select.js"><a href="select.js.html">select.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="8" class="abs high">8/8</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="8" class="abs high">8/8</td>
	</tr>

<tr>
	<td class="file high" data-value="tilesView.js"><a href="tilesView.js.html">tilesView.js</a></td>
	<td data-value="87.88" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 87%"></div><div class="cover-empty" style="width: 13%"></div></div>
	</td>
	<td data-value="87.88" class="pct high">87.88%</td>
	<td data-value="33" class="abs high">29/33</td>
	<td data-value="55" class="pct medium">55%</td>
	<td data-value="20" class="abs medium">11/20</td>
	<td data-value="85" class="pct high">85%</td>
	<td data-value="20" class="abs high">17/20</td>
	<td data-value="96.43" class="pct high">96.43%</td>
	<td data-value="28" class="abs high">27/28</td>
	</tr>

<tr>
	<td class="file medium" data-value="timeGroup.js"><a href="timeGroup.js.html">timeGroup.js</a></td>
	<td data-value="63.33" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 63%"></div><div class="cover-empty" style="width: 37%"></div></div>
	</td>
	<td data-value="63.33" class="pct medium">63.33%</td>
	<td data-value="30" class="abs medium">19/30</td>
	<td data-value="36.67" class="pct low">36.67%</td>
	<td data-value="30" class="abs low">11/30</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="9" class="abs medium">6/9</td>
	<td data-value="65.52" class="pct medium">65.52%</td>
	<td data-value="29" class="abs medium">19/29</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Thu Mar 18 2021 09:42:36 GMT+0100 (GMT+01:00)
            </div>
        </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    