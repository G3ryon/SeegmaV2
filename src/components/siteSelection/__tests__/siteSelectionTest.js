import 'react-native';
import React, { useState } from 'react';

import { act } from 'react-test-renderer';

import { fireEvent, render, waitFor } from '@testing-library/react-native';
import Home from '../home.js';
import { ApplicationProvider, Layout, Button, Text, TopNavigation, TopNavigationAction, IconRegistry } from '@ui-kitten/components';
import { mapping, light as lightTheme } from '@eva-design/eva';
import { EvaIconsPack } from '@ui-kitten/eva-icons';
// Note: test renderer must be required after react-native.
//import renderer from 'react-test-renderer';

jest.useFakeTimers()

test('Login renders correctly', async () => {

    const result = render(
      <SelectionViewTest authToken={"test"} userId={"test"}/>
    );
    await act(async () => {expect(result).toMatchSnapshot();})
  });

  function SelectionViewTest(props) {

    const [testValue0, handleNotification] = useState("noTileValue");
    const [testValue1, setSite] = useState("noIconValue");
    return (
      <>
      <IconRegistry icons={EvaIconsPack} />
      <ApplicationProvider mapping={mapping} theme={lightTheme}>
        <Home {...props} other={props} setNotif={handleNotification} setSite={setSite}/>
      </ApplicationProvider>
      </>
    )
  }