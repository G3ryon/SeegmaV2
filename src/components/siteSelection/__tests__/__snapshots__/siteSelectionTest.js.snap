// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
  >
    <RCTScrollView
      ItemSeparatorComponent={[Function]}
      ListHeaderComponent={
        <TopNavigation
          accessoryLeft={[Function]}
          accessoryRight={[Function]}
          title="Home"
        />
      }
      appearance="default"
      data={null}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={
        Array [
          0,
        ]
      }
      style={
        Array [
          Object {
            "backgroundColor": "#F7F9FC",
          },
          undefined,
        ]
      }
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View>
        <View
          onLayout={[Function]}
        >
          <View
            appearance="default"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFFFFF",
                  "minHeight": 56,
                  "paddingHorizontal": 8,
                  "paddingVertical": 8,
                },
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                },
                undefined,
                undefined,
              ]
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "zIndex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "marginHorizontal": 8,
                    "opacity": 1,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "transform": Array [
                        Object {
                          "scale": 1,
                        },
                      ],
                    }
                  }
                >
                  <RNSVGSvgView
                    align="xMidYMid"
                    bbHeight={24}
                    bbWidth={24}
                    fill="#222B45"
                    focusable={false}
                    meetOrSlice={0}
                    minX={0}
                    minY={0}
                    style={
                      Array [
                        Object {
                          "backgroundColor": "transparent",
                          "borderWidth": 0,
                        },
                        Object {
                          "height": 24,
                          "tintColor": "#222B45",
                          "width": 24,
                        },
                        Object {
                          "flex": 0,
                          "height": 24,
                          "width": 24,
                        },
                      ]
                    }
                    vbHeight={24}
                    vbWidth={24}
                  >
                    <RNSVGGroup
                      fill={4280429381}
                      propList={
                        Array [
                          "fill",
                        ]
                      }
                    >
                      <RNSVGGroup>
                        <RNSVGGroup>
                          <RNSVGRect
                            height="24"
                            matrix={
                              Array [
                                -1,
                                1.2246467991473532e-16,
                                -1.2246467991473532e-16,
                                -1,
                                24,
                                24,
                              ]
                            }
                            opacity={0}
                            width="24"
                            x={0}
                            y={0}
                          />
                          <RNSVGRect
                            height="2"
                            rx=".95"
                            ry=".95"
                            width="18"
                            x="3"
                            y="11"
                          />
                          <RNSVGRect
                            height="2"
                            rx=".95"
                            ry=".95"
                            width="18"
                            x="3"
                            y="16"
                          />
                          <RNSVGRect
                            height="2"
                            rx=".95"
                            ry=".95"
                            width="18"
                            x="3"
                            y="6"
                          />
                        </RNSVGGroup>
                      </RNSVGGroup>
                    </RNSVGGroup>
                  </RNSVGSvgView>
                </View>
              </View>
            </View>
            <View>
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "600",
                      "textAlign": "left",
                    },
                  ]
                }
              >
                Home
              </Text>
            </View>
            <View
              style={
                Array [
                  Object {
                    "flexDirection": "row",
                    "zIndex": 1,
                  },
                  Object {
                    "flex": 1,
                    "justifyContent": "flex-end",
                  },
                ]
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "marginHorizontal": 8,
                    "opacity": 1,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "transform": Array [
                        Object {
                          "scale": 1,
                        },
                      ],
                    }
                  }
                >
                  <RNSVGSvgView
                    align="xMidYMid"
                    bbHeight={24}
                    bbWidth={24}
                    fill="#222B45"
                    focusable={false}
                    meetOrSlice={0}
                    minX={0}
                    minY={0}
                    style={
                      Array [
                        Object {
                          "backgroundColor": "transparent",
                          "borderWidth": 0,
                        },
                        Object {
                          "height": 24,
                          "tintColor": "#222B45",
                          "width": 24,
                        },
                        Object {
                          "flex": 0,
                          "height": 24,
                          "width": 24,
                        },
                      ]
                    }
                    vbHeight={24}
                    vbWidth={24}
                  >
                    <RNSVGGroup
                      fill={4280429381}
                      propList={
                        Array [
                          "fill",
                        ]
                      }
                    >
                      <RNSVGGroup>
                        <RNSVGGroup>
                          <RNSVGRect
                            height="24"
                            opacity={0}
                            width="24"
                            x={0}
                            y={0}
                          />
                          <RNSVGPath
                            d="M20.52 15.21l-1.8-1.81V8.94a6.86 6.86 0 0 0-5.82-6.88 6.74 6.74 0 0 0-7.62 6.67v4.67l-1.8 1.81A1.64 1.64 0 0 0 4.64 18H8v.34A3.84 3.84 0 0 0 12 22a3.84 3.84 0 0 0 4-3.66V18h3.36a1.64 1.64 0 0 0 1.16-2.79zM14 18.34A1.88 1.88 0 0 1 12 20a1.88 1.88 0 0 1-2-1.66V18h4zM5.51 16l1.18-1.18a2 2 0 0 0 .59-1.42V8.73A4.73 4.73 0 0 1 8.9 5.17 4.67 4.67 0 0 1 12.64 4a4.86 4.86 0 0 1 4.08 4.9v4.5a2 2 0 0 0 .58 1.42L18.49 16z"
                          />
                        </RNSVGGroup>
                      </RNSVGGroup>
                    </RNSVGGroup>
                  </RNSVGSvgView>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;
